src/tracks/Pattern.cpp

```diff
@@ -752,8 +752,6 @@ PatternView::~PatternView()
 
 void PatternView::update()
 {
-	m_pat->changeLength( m_pat->length() );
```

Comment:
I do not know the relation to this line of code. Anyway, I cannot reproduce this one bar sequence that reloads as two bars. Is there a sample project that shows the sequence?